doc/tutorial/getting_started.rst

```diff
@@ -11,12 +11,18 @@ Python modules that are not in the standard library:
   that comes with your distribution (e.g. python3-yaml on Debian).
 * `schema`_. Either install via pip (``pip3 install schema``) or the package
   that comes with your distribution (e.g. python3-schema on Debian).
-* `python-magic` Either install via pip (``pip3 install python-magic``) or the package
-  that comes with your distribution.
+* `python-magic`_. Either install via pip (``pip3 install python-magic``) or the package
+  that comes with your distribution (e.g. python3-magic on Debian).
 * `pyparsing`_. Either install via pip (``pip3 install pyparsing``) or the package
   that comes with your distribution (e.g. python3-pyparsing on Debian).
 * gcc
-* `python3-sphinx` Only needed for generating man pages.
+* `python3-sphinx`_. Only needed for generating man pages.
+
+Apart from the tutorial additional dependencies could arise, e.g.:
+
+* curl as the default URL SCM downloader (a limited fallback to wget is possible)
+* extractors based on the supported extensions (e.g. xz, 7z, ...)
+* source code management handlers (e.g. git, svn, ...)
```

Comment:
The generic documentation should be a separate commit. Except the "(a limited fallback to wget is possible)" it's completely unrelated to the change and usefull on it's own.

Reply:
Ok, I will prepare.
How about moving the whole "Getting Started" chapter into the common "Bob User Manual" in between of "Introduction" and "Concepts"? This is more or less the usual place for build-/run time dependencies and installation instructions. I wouldn't search in the tutorial in the first place.